client/web/src/repo/blob/BlameColumn.tsx

```diff
@@ -64,7 +64,7 @@ export const BlameColumn = React.memo<BlameColumnProps>(
             }
 
             const subscription = codeViewElements.subscribe(codeView => {
-                if (codeView) {
+                if (codeView?.firstElementChild) {
                     const table = codeView.firstElementChild as HTMLTableElement
```

Comment:
Nice!

nit: I think we can even move the type assertion to the condition body.

```suggestion
                if (codeView?.firstElementChild instanceOf HTMLTableElement) {
                    const table = codeView.firstElementChild
```